clients/src/main/java/org/apache/kafka/clients/admin/AdminClientConfig.java

```diff
@@ -212,6 +213,7 @@ public class AdminClientConfig extends AbstractConfig {
                                 .define(SECURITY_PROTOCOL_CONFIG,
                                         Type.STRING,
                                         DEFAULT_SECURITY_PROTOCOL,
+                                        in(SecurityProtocol.names().toArray(new String[0])),
```

Comment:
Can we use [Utils::enumOptions](https://github.com/apache/kafka/blob/6d36487b684fd41522cccd4da4fd88f0b89ff0b7/clients/src/main/java/org/apache/kafka/common/utils/Utils.java#L1418-L1433) for this?
```suggestion
                                        in(Utils.enumOptions(SecurityProtocol.class)),
```

Reply:
sure